src/components/crm-grid-table-client/columsGrid.tsx

```diff
@@ -0,0 +1,77 @@
+import { Box } from '@mui/material'
+import { GridColDef } from '@mui/x-data-grid'
+
+import { baseStatusStyle, statusStyles } from './style'
+
+const getStatusStyle = (status: string) => {
+  return {
+    ...baseStatusStyle,
+    ...statusStyles[status]
+  }
+}
+
+const ORDER_STATUS_TRANSLATIONS = {
+  'new': 'Новий',
+  'in processing': 'В обробці',
+  'shipped': 'Відправлено',
+  'delivered': 'Доставлено',
+  'cancelled': 'Скасовано'
```

Comment:
this status should be from be types, @Tkachenko01001  could you show what i mean please

Reply:
![code](https://github.com/user-attachments/assets/021b2681-6e38-4043-8766-9f2822e63e28)

Reply:
Done